src/index.html

```diff
@@ -7,8 +7,81 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+         alt="logo"
+         >
+      <nav class="nav">
+        <ul class="list list__apple">
+          <a class="text text1 is-active"
+            href="#"
+          >
```

Comment:
let's fix it all over your code
```suggestion
          <a class="text is-active" href="#">
```